### PR TITLE
Corrections

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
A few minor corrections on the changes commited to main by @linguist89. 

@linguist89 you can see the file changes in "Files changed" and add comment to those where you might have any. In the top right corner there is a "review changes" where you can add final comments (you don't have to) and then click either "approve", "request changes" depending on whether you think it should be merged.

### CODE REVIEW

File: src/conspiracies/coref/component.py [hunk 1]
```diff
@@ -60,7 +58,7 @@ def __init__(
         if not Span.has_extension("coref_cluster"):
             Span.set_extension("coref_cluster", default=[])
         if not Span.has_extension("antecedent"):
-            Span.set_extension("antecedent", default=Span)
+            Span.set_extension("antecedent", default=[])
```
Comment (linguist89) [inline src/conspiracies/coref/component.py:61]: This shouldn't be a list. If we make it a list then we face the same problem we faced with the clusters i.e. which one in the list is an antecedent? I have made it a span and assigned the first occurring entity in the list of coreference entities as the antecedent.

**Review by linguist89 — COMMENTED** [review]
line 63 shouldn't be a list

File: src/conspiracies/coref/component.py (hunk 1 above)
Comment (KennethEnevoldsen) [reply src/conspiracies/coref/component.py:61]: Then I assume it should be `None` as span is a type object (not an object of type Span).

Comment (KennethEnevoldsen) [inline src/conspiracies/coref/component.py:61]:
```suggestion
            Span.set_extension("antecedent", default=None)
```

**Pull request merged**